src/mlpack/methods/ann/loss_functions/binary_cross_entropy_loss.hpp

```diff
@@ -31,16 +30,16 @@ template <
     typename InputDataType = arma::mat,
     typename OutputDataType = arma::mat
 >
-class CrossEntropyError
+class BCELoss
```

Comment:
We should provide an alias for the class to keep backward compatibility.

Reply:
Could you tell me how to do this.

Reply:
Here is one example:

https://github.com/mlpack/mlpack/blob/56ac3d219b9dc7281ebee5b2385302943070e180/src/mlpack/methods/ann/layer/base_layer.hpp#L272-L278

here we create a alias called `GaussianFunctionLayer` which is basically a `BaseLayer` with the correct template parameter.